examples/smartphone/index.html

```diff
@@ -6,8 +6,7 @@
     <title>Smartphone</title>
     <link rel="stylesheet" type="text/css" href="./css/gsensor.css">
     <script src="https://iottalk.github.io/iottalk-js/jquery.js"></script>
```

Comment:
I think `dan2.js` should only take care of communication part.
BTW, There may be the case that the developer need specific version or doesn't need jQuery.